app/views/site/_study_settings_form.html.erb

```diff
@@ -1,158 +1,57 @@
-<h2>Study Settings</h2>
 <%= nested_form_for(@study, url: update_study_settings_path(accession: @study.accession, study_name: @study.url_safe_name.blank? ? params[:study_name] : @study.url_safe_name), html: {id: 'update-study-settings-form', multipart: true, data: {remote: true}}) do |f| %>
   <% if @study.errors.any? %>
     <div class="bs-callout bs-callout-danger">
       <h4><%= pluralize(@study.errors.count, "error") %> prohibited this study from being saved:</h4>
-
       <ul>
         <% @study.errors.full_messages.each do |message| %>
           <li><%= message %></li>
         <% end %>
       </ul>
     </div>
   <% end %>
-  <div class="form-group row">
-    <div class="col-md-12">
-      <%= f.label :name %><br/>
-      <%= f.text_field :name, class: 'form-control', autocomplete: 'off' %>
-    </div>
-  </div>
-  <div class="form-group row">
-    <div class="col-md-4">
-      <%= f.label :embargo, 'Data release date' %>&nbsp;<span class='fas fa-question-circle' data-toggle='tooltip' title='Setting a data release date will prevent downloads for everyone except study owners & shared users, and will expire at 12AM on the date specified (leave blank to allow downloads)' data-placement='right'></span><br />
-      <%= f.date_field :embargo, class: 'form-control' %>
-    </div>
-    <div class="col-md-4">
-      <%= f.label :public %><br />
-      <%= f.select :public, options_for_select([['Yes',1],['No',0]], @study.public? ? 1: 0), {}, {class: 'form-control'} %>
-    </div>
-    <div class="col-md-4">
-      <%= f.label :cell_count %><br/>
-      <%= f.text_field :cell_count, class: 'form-control' %>
+  <div id="settings-tab-root" class="upload-wizard-react">
+    <div class="col-md-2">
+      <ul class="upload-wizard-steps" role="tablist" id="study-settings-tabs">
+        <li role="presentation" class="settings-nav active" id="settings-general-nav">
+          <a href="#settings-general" data-toggle="tab">General</a>
+        </li>
+        <li role="presentation" class="settings-nav" id="settings-viz-nav">
+          <a href="#settings-viz" data-toggle="tab">View Options</a>
+        </li>
+        <li role="presentation" class="settings-nav" id="settings-sharing-nav">
+          <a href="#settings-sharing" data-toggle="tab">Sharing/Access</a>
+        </li>
+        <li role="presentation" class="settings-nav" id="settings-authors-nav">
+          <a href="#settings-authors" data-toggle="tab">Authors/Publications</a>
+        </li>
+        <li role="presentation" class="settings-nav" id="settings-resources-nav">
+          <a href="#settings-resources" data-toggle="tab">External Resources</a>
```

Comment:
Let's use sentence case for more global consistency across the app.  

We should also generally [avoid slashes in user-facing copy](https://style.mla.org/slashed-terms-mla-style/).  Like how the [GitHub settings UI](https://github.com/settings/profile) uses "and" where a quick note would often use "/" (e.g. "Billing and plans" instead of "Billing/plans"), so we should do here.

```suggestion
          <a href="#settings-viz" data-toggle="tab">View options</a>
        </li>
        <li role="presentation" class="settings-nav" id="settings-sharing-nav">
          <a href="#settings-sharing" data-toggle="tab">Sharing and access</a>
        </li>
        <li role="presentation" class="settings-nav" id="settings-authors-nav">
          <a href="#settings-authors" data-toggle="tab">Authors and publications</a>
        </li>
        <li role="presentation" class="settings-nav" id="settings-resources-nav">
          <a href="#settings-resources" data-toggle="tab">External resources</a>
```